sentry/src/main/java/io/sentry/SentryEnvelopeItemHeader.java

```diff
@@ -14,6 +14,7 @@
   private final @NotNull SentryItemType type;
   private final int length;
   @Nullable private final Callable<Integer> getLength;
+  private final @Nullable String attachmentType;
```

Comment:
as attachmentType had a default value on backend and Android didn't support any other than the default one, we didn't add it, now we need to as values come from 3rd party SDKs eg sentry-native